### DISCUSSION TITLE
The GitHub action is currently failing

### DISCUSSION THREAD

**Discussion by zomars** [snaplet/docs-old#122] [Issues] [answered]
We tried to use the new GitHub action to generate preview DBs and it is throwing this error:

<img width="752" alt="image" src="https://user-images.githubusercontent.com/3504472/200670086-c7be17bb-1645-4e0b-86c2-362fa78f4850.png">

**Comment by jgoux** [comment, marked as answer]
Hey @zomars, nice to see you back!

I think cal.com database server was one of our first, and it might be part of a bug Fly.io had on our earliest machines (it's now fixed).

Could you try deleting the entire server by using `snaplet db clean`? Next time you use `snaplet db create` it will recreate a brand new database server.
